test/JDBC/expected/ISC-Check-Constraints-vu-prepare.out

```diff
@@ -29,8 +29,8 @@ create table test_tsql_cast(
 go
 
 create table test_tsql_collate(
-	c_varchar varchar check(c_varchar <> cast('sflkjasdlkfjf' as varchar(12)) COLLATE bbf_unicode_cp1_ci_as),
```

Comment:
we should have test case for each collation name in reverse transaction table, also we should have test case to make sure that we validate fallback case where transaction table does not find matching TSQL collation

Reply:
We already have a test case where the collation name "japanese_ci_as" doesn't find a match in reverse translation table. 
Should we add test for all the ~20 collation names? We have added tests for first, last value in the table and a negative case where the collation name doesn't exist in the table.

Reply:
we should if we do not have already, specially to guard against table getting messed up in future

Reply:
Added in the next commit.